composer.json

```diff
@@ -13,7 +13,7 @@
   },
   "require": {
     "symfony/yaml": "~2.6",
-    "guzzlehttp/guzzle": "~5.0",
```

Comment:
Its probably just a "bug" from when we moved the cloudflare-plugin-backend out of cPanel and into its own package.  But now that we've removed it from cloudflare-plugin-backend we need it in this repository right?